python/ray/dashboard/modules/job/sdk.py

```diff
@@ -495,9 +496,25 @@ async def tail_job_logs(self, job_id: str) -> AsyncIterator[str]:
             while True:
                 msg = await ws.receive()
```

Comment:
Won’t this ws.receive raise exception when the connection is broken? Isn’t that enough? I actually think there is no need to query job status.